Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -1261,11 +1312,11 @@ theorem eLpNorm_le_mul_eLpNorm_of_ae_le_mul' {f : α → ε} {g : α → ε'} {c
   apply eLpNorm_le_nnreal_smul_eLpNorm_of_ae_le_mul' (h.mono fun _x hx ↦ hx.trans ?_)
   rfl
 
-variable {ε'' : Type*} [TopologicalSpace ε''] [ContinuousENorm ε''] in
+variable {ε : Type*} [TopologicalSpace ε] [ENormedAddMonoid ε] in
```

Comment:
It's hard to check on Github, but can you check that you didn't unnecessarily increased type-class assumptions elsewhere?

Reply:
:heavy_check_mark: